tax-processing-functions/quality_and_split/main.py

```diff
@@ -0,0 +1,403 @@
+"""Copyright 2023 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+# THIS IS JUST DEMO CODE
+
+from datetime import datetime
+import json
+# Import Libraries
+import os
+
+from google.cloud import bigquery
+from google.cloud import documentai_v1 as documentai
```

Comment:
```suggestion
from google.cloud import documentai
```
This automatically defaults to v1